test/index-test.js

```diff
@@ -1,7 +1,44 @@
+const sinon = require('sinon');
 
+describe('Pure functions', function() {
+  describe('getNextEpisodeInPlaylist()', function () {
+    it('should return the next episode in a playlist', function() {
+      const nextEpisode = getNextEpisodeInPlaylist(episodes);
+      expect(nextEpisode).toEqual(episodes[0]);
+    });
+  });
 
-describe('index', function() {
-  it('runs', function() {
-    expect(true).toEqual(true)
-  })
-})
+  describe('addToPlaylist()', function () {
+    it('should add a given episode to a given playlist', function() {
+      const updatedPlaylist = addToPlaylist(episodes, finaleEpisode);
+      expect(updatedPlaylist.length).toEqual(episodes.length + 1);
+    });
+
+    it('should not modify the original episodes array', function() {
+      addToPlaylist(episodes, finaleEpisode);
+      expect(episodes.length).toEqual(4);
+    });
+  });
+
+  describe('removeFromPlaylist()', function () {
+    it('should remove a given episode from a given playlist', function() {
+      const updatedPlaylist = removeFromPlaylist(episodes, episodes[0]);
+      expect(updatedPlaylist.length).toEqual(episodes.length - 1);
+    });
+
+    it('should not modify the original episodes array', function() {
+      removeFromPlaylist(episodes, episodes[0]);
+      expect(episodes.length).toEqual(4);
+    });
+  });
+
+  describe('bingeWatch()', function () {
+    it('should watch all 5 episodes', function() {
+      const updatedPlaylist = addToPlaylist(episodes, finaleEpisode);
+      const spy = sinon.spy(window, 'getNextEpisodeInPlaylist');
```

Comment:
This spy won't work in the browser (index.html) because `getNextEpisodeInPlaylist` won't be attached to `window` (since the solution has it declared with `const`). We might want to flag that for students :/